module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -494,6 +494,21 @@ public function getConfig($function, $params = [])
         return $this->config[$function] ?? false;
     }
 
+    /**
+     * Check whether an item is holdable based on its location and any
+     * current loan
+     *
+     * @param string         $locationName locationName from getHolding
+     * @param \stdClass|void $currentLoan  The current loan, or null if none
```

Comment:
Should this be `?\stdClass` rather than `\stdClass|void`?

Reply:
Yes probably. :)  I can't find the `?` syntax documented?
https://docs.phpdoc.org/guide/references/phpdoc/tags/param.html
https://docs.phpdoc.org/guide/guides/types.html

Reply:
It's a native PHP feature -- see "Nullable type syntactic sugar" in the [PHP manual](https://www.php.net/manual/en/language.types.declarations.php). I assume that what's legal in PHP is legal in PHPDoc, but maybe I'm mistaken. If so, we have a lot of errors in our existing documentation.

In any case, even if using the `?` syntax is not ideal, we definitely should not use `void` in this context -- it is only for return values, and nullable parameters should use `null` instead.